blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -33,6 +37,7 @@ sealed abstract class BlazeClientBuilder[F[_]] private (
     val parserMode: ParserMode,
     val bufferSize: Int,
     val executionContext: ExecutionContext,
+    val scheduler: Option[TickWheelExecutor],
```

Comment:
I think I like the [Ember idiom for defaultable resources](https://github.com/http4s/http4s/blob/05501d7f07481ead542de44e516f693f7d32a673/ember-client/src/main/scala/org/http4s/ember/client/EmberClientBuilder.scala#L56), where the builder provides a default, or an unmanaged one can be passed in.